ci/cloudbuild/dockerfiles/ubuntu-bionic-install.Dockerfile

```diff
@@ -104,7 +104,7 @@ RUN curl -sSL https://github.com/google/crc32c/archive/1.1.0.tar.gz | \
       -DCRC32C_BUILD_BENCHMARKS=OFF \
       -DCRC32C_USE_GLOG=OFF \
       -H. -Bcmake-out -GNinja && \
-    cmake --build cmake-out --target install && \
+    cmake --build cmake-out --target install -- -j ${NCPU:-4} && \
```

Comment:
I thought you wanted "${NCPU}" in non-demo files?

Reply:
Oops. Fixed. Re-testing.